packages/ui/src/multi-select.tsx

```diff
@@ -134,23 +136,14 @@ export const MultiSelect = React.forwardRef<HTMLButtonElement, MultiSelectProps>
 		return (
 			<Popover
 				open={isPopoverOpen}
-				// open={isPopoverOpen}
-				// onOpenChange={(open) => {
-				// 	console.log("GOING TO", open);
-				// 	setIsPopoverOpen(open);
-				// }}
 				onOpenChange={(open) => {
-					if (isPopoverOpen && !open) {
-						return;
-					}
```

Comment:
this was making the popover not close when clicking outside of it. I'm not sure if removing this breaks something else though... I played around and didn't find anything though

Reply:
!!! (someone who knows should comment but omg, tyty, this wastes so much of my time)